src/organizer.js

```diff
@@ -63,7 +63,13 @@ class Organizer {
           console.log(`send successful for ${JSON.stringify(userA)} and ${JSON.stringify(userB)}`);
           return this.recordMeeting(userA, userB);
         })
-        .catch(() => {
+        .catch((error) => {
+          if (error && error.response && error.response.data) {
+            const err = error.response.data;
```

Comment:
Nitpick: `err` out of `error.response.data` looks a bit unusual. How about using some destructuring assignment?

```js
const { StatusCode, ErrorMessage } = error.response.data;
```

It looks a bit cleaner and you can skip the long chaining that you're trying to avoid.